src/style.css

```diff
@@ -1 +1,81 @@
 /* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+  background-color: aliceblue;
+  margin: 0;
+  padding: 0;
```

Comment:
there is no default padding for body, so you can remove it

Reply:
* Not fixed: